### PR TITLE
Feature/event editor

### PR DESCRIPTION
**Pull request by yscope75**
@nguyenductoan  please review this !

### CODE REVIEW

File: app/controllers/event_editor/event_editors_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#  service                :string
+#  rating                 :integer
+#  partner_id             :integer
+#  student_type           :string
+#
+ 
+class EventEditor::EventEditorsController < ApplicationController
+  before_action :authenticate_user!
+  layout "event_editor"
+  def ensure_partner
```
Comment (nguyenductoan) [inline app/controllers/event_editor/event_editors_controller.rb]: why did you check current_user if that event_editor or not, but your method's name is ensure_partner?

File: app/controllers/event_editor/events_controller.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      format.html { redirect_to event_editor_events_url, notice: 'event was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
+  private
+  def set_event
+    @event = Event.find(params[:id])
+  end
+  def event_params
+    params.require(:event).permit(:name, :content)
```
Comment (nguyenductoan) [inline app/controllers/event_editor/events_controller.rb]: Please ask customer about information that event need to have before create model.

File: app/controllers/event_editor/event_editors_controller.rb (hunk 1 above)
Comment (yscope75) [inline app/controllers/event_editor/event_editors_controller.rb]: Thanks @nguyenductoang, i was not planning to use it and forgot delete. I will fix it now !

**Comment by lhhieu89** [comment]
@yscope75 : Please fix the conflict, thanks

**Comment by yscope75** [comment]
@lhhieu89  i fixed it, please check !

**Comment by lhhieu89** [comment]
@yscope75 : Ok thank you

File: app/controllers/event_editor/event_editors_controller.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+#  status                 :string
+#  service                :string
+#  rating                 :integer
+#  partner_id             :integer
+#  student_type           :string
+#
+ 
+class EventEditor::EventEditorsController < ApplicationController
+  before_action :authenticate_user!
+  layout "event_editor"
```
Comment (lhhieu89) [inline app/controllers/event_editor/event_editors_controller.rb]: Is it the same partner's layout?

File: app/views/event_editor/events/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.box-header
+  = link_to 'Back', event_editor_events_path, class: 'btn btn-flat btn-primary'
+
+.box 
+  .box-header.with-border
+    h1.box-title Edit Student 
+  .box-body
+    == render 'form'
```
Comment (lhhieu89) [inline app/views/event_editor/events/edit.html.slim]: What is `==` mean?

File: app/models/user.rb [hunk 5]
```diff
@@ -42,8 +43,8 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
   #has_one :profile, :dependent => :destroy, :inverse_of => :user
   #accepts_nested_attributes_for :profile, :allow_destroy => false
-  #validates :profile, presence: true
-  enum type: { admin: 'Admin', partner: 'Partner', student: 'Student' }
+  #validates :profile, presence: trues
```
Comment (lhhieu89) [inline app/models/user.rb]: What is `trues`, please remove it

File: app/controllers/event_editor/event_editors_controller.rb (hunk 3 above)
Comment (yscope75) [inline app/controllers/event_editor/event_editors_controller.rb]: it different from partner's layout

File: app/controllers/application_controller.rb [hunk 6]
```diff
@@ -8,6 +8,8 @@ def after_sign_in_path_for(resource)
   		admin_dashboard_index_path
   	elsif current_user.partner?
       partner_path
+    elsif current_user.event_editor?
+      event_editor_events_path
     else
       root_path
   	end
```
Comment (lhhieu89) [inline app/controllers/application_controller.rb]: Please reindent the code in this file

**Comment by yscope75** [comment]
Thanks @lhhieu89  i will fix now !

**Pull request merged**